src/options/Makefile.am

```diff
@@ -22,191 +8,51 @@
-	proof_options.tmp \
-	prop_options.tmp \
-	quantifiers_options.tmp \
-	sep_options.tmp \
-	sets_options.tmp \
-	smt_options.tmp \
-	strings_options.tmp \
-	theory_options.tmp \
-	uf_options.tmp
-
-OPTIONS_OPTIONS_FILES = \
-	arith_options.options \
-	arrays_options.options \
-	base_options.options \
-	booleans_options.options \
-	builtin_options.options \
-	bv_options.options \
-	datatypes_options.options \
-	decision_options.options \
-	expr_options.options \
-	fp_options.options \
-	idl_options.options \
-	main_options.options \
-	parser_options.options \
-	printer_options.options \
-	proof_options.options \
-	prop_options.options \
-	quantifiers_options.options \
-	sep_options.options \
-	sets_options.options \
-	smt_options.options \
-	strings_options.options \
-	theory_options.options \
-	uf_options.options
-
-OPTIONS_SEDS = \
-	arith_options.sed \
-	arrays_options.sed \
-	base_options.sed \
-	booleans_options.sed \
-	builtin_options.sed \
-	bv_options.sed \
-	datatypes_options.sed \
-	decision_options.sed \
-	expr_options.sed \
-	fp_options.sed \
-	idl_options.sed \
-	main_options.sed \
-	parser_options.sed \
-	printer_options.sed \
-	proof_options.sed \
-	prop_options.sed \
-	quantifiers_options.sed \
-	sep_options.sed \
-	sets_options.sed \
-	smt_options.sed \
-	strings_options.sed \
-	theory_options.sed \
-	uf_options.sed
-
-OPTIONS_HEADS = \
-	arith_options.h \
-	arrays_options.h \
-	base_options.h \
-	booleans_options.h \
-	builtin_options.h \
-	bv_options.h \
-	datatypes_options.h \
-	decision_options.h \
-	expr_options.h \
-	fp_options.h \
-	idl_options.h \
-	main_options.h \
-	parser_options.h \
-	printer_options.h \
-	proof_options.h \
-	prop_options.h \
-	quantifiers_options.h \
-	sep_options.h \
-	sets_options.h \
-	smt_options.h \
-	strings_options.h \
-	theory_options.h \
-	uf_options.h
-
-OPTIONS_CPPS = \
-	arith_options.cpp \
-	arrays_options.cpp \
-	base_options.cpp \
-	booleans_options.cpp \
-	builtin_options.cpp \
-	bv_options.cpp \
-	datatypes_options.cpp \
-	decision_options.cpp \
-	expr_options.cpp \
-	fp_options.cpp \
-	idl_options.cpp \
-	main_options.cpp \
-	parser_options.cpp \
-	printer_options.cpp \
-	proof_options.cpp \
-	prop_options.cpp \
-	quantifiers_options.cpp \
-	sep_options.cpp \
-	sets_options.cpp \
-	smt_options.cpp \
-	strings_options.cpp \
-	theory_options.cpp \
-	uf_options.cpp
-
+OPTIONS_CONFIG_FILES = \
+	arith_options.toml \
+	arrays_options.toml \
+	base_options.toml \
+	booleans_options.toml \
+	builtin_options.toml \
+	bv_options.toml \
+	datatypes_options.toml \
+	decision_options.toml \
+	expr_options.toml \
+	fp_options.toml \
+	idl_options.toml \
+	main_options.toml \
+	parser_options.toml \
+	printer_options.toml \
+	proof_options.toml \
+	prop_options.toml \
+	quantifiers_options.toml \
+	sep_options.toml \
+	sets_options.toml \
+	smt_options.toml \
+	strings_options.toml \
+	theory_options.toml \
+	uf_options.toml
+
+OPTIONS_HEADS = $(OPTIONS_CONFIG_FILES:.toml=.h)
```

Comment:
OPTIONS_HEADERS

Reply:
That was actually the issue with the failing autogen.sh. something_HEADERS is autotools magic :-)